app/ai/voice/agents/breeze_buddy/managers/calls.py

```
@@ -467,6 +467,40 @@ async def handle_unanswered_calls(call_id: str):
+                    webhook_data,
+                    max_retries=3,
+                )
+                if success:
+                    logger.info(
+                        f"Successfully sent webhook for unanswered call {call_id}"
+                    )
+                else:
+                    logger.error(
+                        f"Failed to send webhook for unanswered call {call_id} after all retries"
+                    )
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)